### PR TITLE
Fixing retrieving metadata examples

### PR DESCRIPTION
**Pull request by carherco**
There was no example about retrieving place metadata from the controller. I tried to fixed it as well as the example of retrieving transition metadata. Don't know wether the code I propose is correct. It would need a revision from anybody who knows how it works.

### CODE REVIEW

File: workflow.rst [hunk 1]
```diff
@@ -612,21 +612,24 @@ Then you can access this metadata in your controller as follows::
         ;
 
         // or
-        $title = $workflow->getMetadataStore()
-            ->getWorkflowMetadata()['title'] ?? false
+        $max_num_of_words = $workflow->getMetadataStore()
```
Comment (javiereguiluz) [inline workflow.rst]: For PHP variables we use `$camelCase`, so please update this and the other occurrence to `$maxNumOfWords`. Thanks! Note: we can use `max_num_of_words` as the PHP array key.

Comment (carherco) [reply workflow.rst]: PHP variable name changed. 
The PHP array key name comes from the configuration example (line 528).
Thanks!

File: workflow.rst [hunk 2]
```diff
@@ -612,21 +612,24 @@ Then you can access this metadata in your controller as follows::
         ;
 
         // or
-        $title = $workflow->getMetadataStore()
-            ->getWorkflowMetadata()['title'] ?? false
+        $maxNumOfWords = $workflow->getMetadataStore()
+            ->getPlaceMetadata('draft')['max_num_of_words'] ?? false
```
Comment (noniagriconomie) [inline workflow.rst]: instead of false, what about an integer? so we stay in the same type

File: workflow.rst [hunk 3]
```diff
@@ -612,21 +612,24 @@ Then you can access this metadata in your controller as follows::
...
         ;
 
         // or
         $aTransition = $workflow->getDefinition()->getTransitions()[0];
-        $transitionTitle = $workflow
+        $priority = $workflow
             ->getMetadataStore()
-            ->getTransitionMetadata($aTransition)['title'] ?? false
+            ->getTransitionMetadata($aTransition)['priority'] ?? false
```
Comment (noniagriconomie) [inline workflow.rst]: same

**Comment by noniagriconomie** [comment]
@carherco can you check if your Pr is still relevant on 4.3 doc now ?
Thx

**Comment by carherco** [comment]
@noniagriconomie MetadataStoreInterface (https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Workflow/Metadata/MetadataStoreInterface.php) has not changed in 4.3 so this docs should be the same as in 4.2 version.

File: workflow.rst (hunk 3 above)
Comment (carherco) [reply workflow.rst]: solved ;-)

**Comment by noniagriconomie** [comment]
hello @carherco , i meant rebase the PR and see if your addition is still relevant as it is or if it needs rewrite :)

**Comment by carherco** [comment]
Sorry, I don't manage well rebase, merge, pull request...

I merged branch 'master' into patch-1, repair conflicts and rewrite some docs.

Then I realised that you asked me to do it not with 'master' but with branch 4.3. I don't know how, but I did a pull request: https://github.com/symfony/symfony-docs/pull/11591

I don't know if i am doing things well or not. I hope you are able to manage with the mess I created. Sorry.

**Review by lyrixx — APPROVED** [review]
Perfect :+1: 
Thanks

**Comment by javiereguiluz** [comment]
Closed in favor of #12360. Thanks!

**Pull request closed without merging**